src/components/switch/switch.stories.mdx

```diff
@@ -367,24 +367,24 @@ It is possible to use the `tooltipPosition` to override the default placement of
 
 ## Translation keys
 
-The following keys are available to override the translations for this component by passing in a custom locale object to the 
+The following keys are available to override the translations for this component by passing in a custom locale object to the
 [i18nProvider](https://carbon.sage.com/?path=/story/documentation-i18n--page).
 
 <TranslationKeysTable
-  translationData={
-    [
-      {
-        name: "switch.on",
-        description: "The text fto display o indicate action would move component to on state",
-        type: "func",
-        returnType: "string"
-      },
-      {
-        name: "switch.off",
-        description: "The text to display to indicate action would move component to off state",
-        type: "func",
-        returnType: "string"
-      },
-    ]
-  }
+  translationData={[
```

Comment:
This should be in separate `docs` commit.

Reply:
changed, thanks